src/components/Nav/Nav.js

```diff
@@ -35,64 +48,104 @@ function Nav() {
   const isCreateAccount = location.pathname === '/createaccount';
   const isLogin = !!localStorage.getItem('token');
 
+  useEffect(() => {
+    searchInput ? history.push(`/?search=${searchInput}`) : history.push('/');
+  }, [searchInput]);
```

Comment:
```suggestion
  useEffect(() => {
    searchInput
      ? history.push(`/?search=${searchInput}`)
      : history.push('/');
  }, [searchInput]);
```
- 이동하려는 경로가 잘 보이도록 줄바꿈 해주는 것도 좋을거 같은데 린트가 허락해주지 않는다면 이대로 가시죠!
- but, state로 관리할 필요가 있는지 검토 해주세요

Reply:
린트가 허락해주지 않습니다!